.gitmodules

```diff
@@ -0,0 +1,3 @@
+[submodule "yoga"]
+	path = yoga
+	url = git@github.com:facebook/yoga.git
```

Comment:
Please use `https://github.com/facebook/yoga.git` as the git URL so that we have to have github get feisty with SSH permissions.